### PR TITLE
Pass default offerConstraints to simple peer

### PR DESCRIPTION
**Pull request by bthesorceror**
Fixes an issue with using simple-peer needing offerConstraints with wrtc

### CODE REVIEW

**Comment by perguth** [comment]
We should rather pass [offerConstraints](https://github.com/feross/simple-peer/search?utf8=%E2%9C%93&q=offerConstraints) through instead of just setting it to an empty object.

**Comment by bthesorceror** [comment]
I have updated this to allow someone using the library to pass in offer constraints when they create a swarm or to use a default of an empty object

File: index.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ module.exports = function (hub, opts) {
   if (!opts) opts = {}
   var wrap = opts.wrap || function (data) { return data }
   var unwrap = opts.unwrap || function (data) { return data }
+  var offerConstraints = opts.offerConstraints || {};
```
Comment (perguth) [inline index.js]: One semicolon too much (see [standard](https://github.com/feross/standard)).

**Comment by bthesorceror** [comment]
Semi colon removed to conform to standard :)

**Comment by perguth** [comment]
🍰

**Comment by mafintosh** [comment]
2.5.0

**Pull request merged**